cmake/modules/RootNewMacros.cmake

```diff
@@ -1079,6 +1079,22 @@ function(ROOT_ADD_TEST test)
 endfunction()
 
 #----------------------------------------------------------------------------
+# function ROOT_ADD_GTEST(<testsuite> <name> LIBRARIES)
+#
+function(ROOT_ADD_GTEST test_suite test_name)
+  CMAKE_PARSE_ARGUMENTS(ARG "" "" "LIBRARIES" ${ARGN})
+
+  include_directories(${GTEST_INCLUDE_DIR} ${GMOCK_INCLUDE_DIR})
```

Comment:
Can we add the source directory to the include dirs, or does that happen by default? (I want to be able to `#include` a header in the tests.)